crates/red_knot/src/types.rs

```diff
@@ -317,9 +317,11 @@ impl ModuleTypeStore {
         }
     }
 
-    fn add_class(&mut self, name: &str) -> ClassTypeId {
+    fn add_class(&mut self, name: &str, bases: &[Type]) -> ClassTypeId {
```

Comment:
We should take a `Vec<Type>` here, same for the other `add_class` function. 

I use a slice when I only need to read the data, need to transform the data, or only sometimes need to copy the data but I use a owned type if the function always needs to convert type to an owned type. That gives the control to the caller, maybe the caller already has an owned type (which is the case here), and converting it to a slice, and then copying to a vec is a lot of unnecessary work.

Reply:
Makes sense! I'll change this.